content/uk/docs/contribute/localization_uk.md

```diff
@@ -0,0 +1,123 @@
+
+* Частовживані і усталені за межами K8s слова перекладаємо українською і пишемо з маленької літери (label -> мітка).
+
+* Для перевірки закінчень слів у родовому відмінку однини (-а/-я, -у/-ю) використовуйте [онлайн словник](https://slovnyk.ua/). Якщо слова немає у словнику, визначте його відміну і далі відмінюйте за правилами. Більшість необхідних нам термінів є словами іншомовного походження, які у родовому відмінку однини приймають закінчення -а: Пода, Deployment'а. Докладніше [дивіться тут](https://pidruchniki.com/1948041951499/dokumentoznavstvo/vidminyuvannya_imennikiv).
+
+## Словник {#словник}
+
+English | Українська |
+--- | --- |
+addon | розширення |
+application | застосунок |
+backend | бекенд |
+build | збирати (процес) |
+build | збирання (результат) |
+cache | кеш |
+CLI | інтерфейс командного рядка |
+cloud | хмара; хмарний провайдер |
+containerized | контейнеризований |
+Continuous development | безперервна розробка |
+Continuous integration | безперервна інтеграція |
+Continuous deployment | безперервне розгортання |
+contribute | робити внесок (до проекту), допомагати (проекту) |
+contributor | контриб'ютор, учасник проекту |
+control plane | площина управління |
+controller | контролер |
+CPU | ЦПУ |
+dashboard | дашборд |
+data plane | площина даних |
+default settings | типові налаштування |
+default (by) | за умовчанням |
+Deployment | Deployment |
+deprecated | застарілий |
+desired state | бажаний стан |
+downtime | недоступність, простій |
+ecosystem | сімейство проектів (екосистема) |
+endpoint | кінцева точка |
+expose (a service) | відкрити доступ (до сервісу) |
+fail | відмовити |
+feature | компонент |
+framework | фреймворк |
+frontend | фронтенд |
+image | образ |
+Ingress | Ingress |
+instance | інстанс |
+issue | запит |
+kube-proxy | kube-proxy |
+kubelet | kubelet |
+Kubernetes features | функціональні можливості Kubernetes |
+label | мітка |
+lifecycle | життєвий цикл |
+logging | логування |
+maintenance | обслуговування |
+master | master |
+map | спроектувати, зіставити, встановити відповідність |
+monitor | моніторити |
+monitoring | моніторинг |
+Namespace | Namespace |
+network policy | мережева політика |
+node | вузол |
+orchestrate | оркеструвати |
+output | вивід |
+patch | патч |
+Pod | Под |
+production | прод |
+pull request | pull request |
+release | реліз |
+replica | репліка |
+rollback | відкатування |
+rolling update | послідовне оновлення |
+rollout (new updates) | викатка (оновлень) |
+run | запускати |
+scale | масштабувати |
+schedule | розподіляти (Поди по вузлах) |
+scheduler | scheduler |
+secret | секрет |
+selector | селектор |
+self-healing | самозцілення |
+self-restoring | самовідновлення |
+service | сервіс |
+service discovery | виявлення сервісу |
+source code | вихідний код |
+stateful app | застосунок зі станом |
+stateless app | застосунок без стану |
+task | завдання |
+terminated | зупинений |
+traffic | трафік |
+VM (virtual machine) | ВМ |
+Volume | Volume |
```

Comment:
nit:
```suggestion
volume | volume |
```

Reply:
Based on this
https://github.com/kubernetes/website/pull/20020/files#diff-11dd086c148eeba321233dba5426004eR29

I propose in glossary use same letters case if word will be not at the beginning of the sentence. If it is not obvious, I can add clarification in doc.

Also, thanks, I need carefully re-review glossary to fix letter case in words like `Service`